<div class="chatlog__message-group">
  <div id="chatlog__message-container-309165835828723712" class="chatlog__message-container " data-message-id="309165835828723712">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <img class="chatlog__avatar" src="https://cdn.discordapp.com/avatars/293572563010060288/92f572b5e1ef6656a55498787c15a5ad.png?size=512" alt="Avatar" loading="lazy">
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__header">
          <span class="chatlog__author" style="color: rgb(231, 76, 60)" title="tarunon#9603" data-user-id="293572563010060288">tarunon</span>
          <a href="/channels/291054454793306112?category=main&amp;channel=swift&amp;message_id=309166576597336067"><span class="chatlog__timestamp">03-May-17 03:14 AM</span></a>
        </div>
        <div class="chatlog__content chatlog__markdown">
          <span class="chatlog__markdown-preserve">asyncとthrowsを組み合わせた時に何が起こるのかが解りにくい気がします</span>
        </div>
      </div>
    </div>
  </div>
  <div id="chatlog__message-container-309166576597336067" class="chatlog__message-container " data-message-id="309166576597336067">
    <div class="chatlog__message">
      <div class="chatlog__message-aside">
        <div class="chatlog__short-timestamp" title="03-May-17 03:17 AM">03:17</div>
      </div>
      <div class="chatlog__message-primary">
        <div class="chatlog__content chatlog__markdown">
          <span class="chatlog__markdown-preserve">1.&nbsp;入れ子関係は固定なのか&nbsp;(必ずasyncが外側? 2.&nbsp;Errorの扱いがどうなるのか (↑の通りなら&nbsp;() async -&gt; Error になっている? 3. waitとcatchの併用が可能なのか、併用したときのErrorの扱いについて</span>
        </div>
      </div>
    </div>
  </div>
</div>